src/smart-app-banner/smart-app-banner.tsx

```diff
@@ -23,7 +23,7 @@ interface SmartAppBannerProps {
     onBannerPress?: () => void | undefined;
 }
 
-interface Content {
+export interface Content {
```

Comment:
suggest renaming to something like `SmartAppBannerContent` to avoid naming collision

Reply:
To rename as `SmartAppBannerContentProps`